### PR TITLE
Updating some Video Compression stuff

### PR DESCRIPTION
**Pull request by wawesomeNOGUI**
Hi! Just doin some more compression learning. Do you think saying “like the JPEG compression method.” Is good?

### CODE REVIEW

File: content/docs/06-media-communication.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ These constraints are all caused by the limitations of the real world. They are
 Transporting video isn't easy. To store 30 minutes of uncompressed 720 8-bit video you need ~110Gb. With those numbers a 4 person conference call isn't going to happen. We need a way to make it smaller, and the answer is video compression. That doesn't come without downsides though.
 
 ## Video 101
-We aren't going to cover video compression in depth, but just enough to understand why RTP is designed the way it is. Video compression encodes video into a new format that requires fewer bits.
+We aren't going to cover video compression in depth, but just enough to understand why RTP is designed the way it is. Video compression encodes video into a new format that requires fewer bits to represent the same video, usually with a slight loss in quality.
```
Comment (Sean-Der) [inline content/docs/06-media-communication.md:31]: What do you think of mentioning that this is up to the encoder? You can choose to be lossy or lossless. You can also choose how much loss in quality you are willing to tolerate (in exchange for less space or CPU usage?)

File: content/docs/06-media-communication.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ If set, the RTP header will have extensions. This is described in greater detail
 The amount of `CSRC` identifiers that follow after the `SSRC`, and before the payload.
 
 #### Marker (M)
-The marker bit has no pre-set meaning, and is up to the user.
+The marker bit has no pre-set meaning, and can be used however the user likes.
```
Comment (Sean-Der) [inline content/docs/06-media-communication.md:91]: :+1:

File: content/docs/06-media-communication.md [hunk 3]
```diff
@@ -28,13 +28,13 @@ These constraints are all caused by the limitations of the real world. They are
...
 ## Video 101
-We aren't going to cover video compression in depth, but just enough to understand why RTP is designed the way it is. Video compression encodes video into a new format that requires fewer bits.
+We aren't going to cover video compression in depth, but just enough to understand why RTP is designed the way it is. Video compression encodes video into a new format that requires fewer bits to represent the same video, usually with a slight loss in quality.
 
 ### Lossy and Lossless compression
 It can be lossless (no information is lost) or lossy (information may be lost). RTP typically uses lossy compression. It is better to have some details lost in the picture, then have it not arrive at all.
 
 ### Intra and Inter frame compression
-Video compression then comes in two types. The first is intra-frame. This is find a way to compress a single picture, the same techniques that are used in things like JPEG.
+Video compression comes in two types. The first is intra-frame. Intra-frame compression reduces the bits used to describe a single video frame. The same techniques are used to compress still pictures, like the JPEG compression method.
```
Comment (Sean-Der) [inline content/docs/06-media-communication.md:37]: :+1:

File: content/docs/06-media-communication.md (hunk 1 above)
Comment (wawesomeNOGUI) [reply content/docs/06-media-communication.md:31]: That sounds good! I’ll write some more in there

Comment (wawesomeNOGUI) [reply content/docs/06-media-communication.md:31]: Does it sound good to shorten that sentence, and instead add more under the Lossy versus Lossless header?

Comment (wawesomeNOGUI) [reply content/docs/06-media-communication.md]: Cool! Changed some stuff, I wasn’t sure if lossy encoding or lossless encoding took more cpu usage?

**Review by mogren — APPROVED** [review]

**Pull request merged**